numba/cuda/testing.py

```diff
@@ -30,6 +30,10 @@ def skip_unless_conda_cudatoolkit(reason):
     return unittest.skipUnless(get_conda_ctk() is not None, reason)
 
 
+def skip_with_external_memmgr(reason):
```

Comment:
Docstring? Also, should it be called `skip_if_external_memmgr` or `skip_on_external_memmgr` maybe?

Reply:
There's no docstring for any skip method in `numba.cuda.testing`, but I can add them. I'll go with `skip_if_external_memmgr`, which I think sounds better.

Reply:
Renamed to `skip_if_external_memmgr` and added docstrings.